supplements/critical-role-call-of-the-netherdeep/items.xml

```diff
@@ -0,0 +1,624 @@
+		<compendium display="false" />
+		<description>
+			<p>The <i>Jewel of Three Prayers</i> is a Vestige of Divergence (see the "Vestiges of Divergence" sidebar). In ancient times, Alyxian the Apotheon bore this amulet as a symbol of his covenant with three Prime Deities: Sehanine the Moon Weaver, Avandra the Change Bringer, and Corellon the Arch Heart. When the jewel is found, only Sehanine's power thrums within its dormant heart. The power of the other two deities waits to be reawakened by a hero—or heroes—who can follow in Alyxian's footsteps.</p>
+			<p class="indent"><strong><em>Dormant State.</em></strong> In this state, the <i>Jewel of Three Prayers</i> is a glittering golden disk attached to a fine golden chain. The chain magically resizes to function as a necklace for the creature that wears it.</p>
+			<p class="indent">In its Dormant State, the jewel has the following properties:</p>
+			<ul>
+				<li>You gain a +1 bonus to AC while wearing the jewel.</li>
+				<li>While wearing or holding the jewel you can use an action to cause it to shed bright light in a 15-foot radius and dim light for an additional 15 feet. The light lasts until you extinguish It (no action required).</li>
+				<li>The jewel has 3 charges and regains all its expended charges daily at dawn. While holding the jewel, you can expend 1 charge from it to cast the <i>invisibility</i> spell.</li>
+			</ul>
+			<div class="sidebar">
+				<h5>VESTIGES OF DIVERGENCE</h5>
+				<p>When the great wars of the Calamity rolled across Exandria, heroes of divinity accepted legendary relics from their patrons and gods. At the same time, archmages wove dangerous arcane power into items of immense magical might. When the war ended, most of these magic items were buried with their wielders beneath ash and dust. Those items that remained accessible were passed down as heirlooms. These remnants of a lost era are known as the Vestiges of Divergence</p>
+				<p class="indent">Each Vestige of Divergence is a magic item that grows in power over time. When it is first encountered, the item is usually in its Dormant State. Either on its own or in response to the heroic deeds of its bearer, a Vestige of Divergence can enter its Awakened State. In time, it can reach its ultimate potential and enter its Exalted State, usually in response to a momentous event or some heroic achievement on the part of its bearer. The properties gained at each new state are cumulative with the properties of a previous state, unless otherwise noted.</p>
+			</div>
+		</description>
+		<setters>
+			<set name="category">Wondrous Items</set>
+			<set name="type">Wondrous Item</set>
+			<set name="rarity">Legendary</set>
+			<set name="keywords">vestige of divergence, ac, invisibility</set>
+			<set name="attunement">true</set>
+			<set name="charges">3</set>
+			<set name="cost" currency="gp">0</set>
```

Comment:
```suggestion
			<set name="cost" currency="gp">0</set>
			<set name="slot">misc</set>
```
There is a bug with items that require attunement and are not equip-able that sometimes the effect is kept even without the items.  The app also lags more without it